cli/cli

```diff
@@ -181,6 +181,7 @@ if [ "$COMMAND" == "provision" ] ; then
   docker cp ${SCRIPTS}/. ${VM_CID}:/scripts/
   docker exec ${VM_CID} /bin/bash -c "while [ ! -f /usr/local/bin/ssh.sh ] ; do sleep 1; done"
   docker exec ${VM_CID} /bin/bash -c "ssh.sh sudo /bin/bash -s ${CRIO} < /scripts/provision.sh"
+  docker exec ${VM_CID} /bin/bash -c "ssh.sh sudo /bin/bash < /logging.sh"
```

Comment:
As said above, I would prefer if we don't do anything regarding to logging here, I would want it to be an explicit decision by the users. For instance if we don't expose a log-dir, we may pile up gigabytes of logs quickly and no one knows what is going on.

Reply:
> The solution I think could work, is to provide after run deployment of a fluent daemon set, which would be default, but can be easily overridden.

@petrkotas ok, let's see where that leads us.